includes/Queue.h

```diff
@@ -16,9 +16,13 @@ enum EventType {
 
 struct Event {
   Event(const EventType type, const std::string &directory, const std::string &fileA, const std::string &fileB) :
-      type(type), directory(directory), fileA(fileA), fileB(fileB) {}
+      type(type), directory(directory), fileA(fileA), fileB(fileB) {
+    timePoint = std::chrono::high_resolution_clock::now();
```

Comment:
rename `timePoint` to the more idiomatic name `timestamp`, please

Reply:
with forgiveness I think it is a time point because the std::chrono libraries doesn't know any 'timestamps' just a lot of time points

see: http://en.cppreference.com/w/cpp/chrono/time_point
"Class template std::chrono::time_point represents a point in time. It is implemented as if it stores a value of type Duration indicating the time interval from the start of the Clock's epoch."

if we rename this as "timestamp" there is confusion with some known timestamps